app/controllers/hyrax/depositors_controller.rb

```diff
@@ -38,8 +38,8 @@ def authorize_and_return_grantor
       def send_proxy_depositor_added_messages(grantor, grantee)
         message_to_grantee = "#{grantor.name} has assigned you as a proxy depositor"
         message_to_grantor = "You have assigned #{grantee.name} as a proxy depositor"
-        ::User.batch_user.send_message(grantor, message_to_grantor, "Proxy Depositor Added")
-        ::User.batch_user.send_message(grantee, message_to_grantee, "Proxy Depositor Added")
+        Hyrax::MessengerService.deliver(::User.batch_user, grantor, message_to_grantor, "Proxy Depositor Added")
```

Comment:
I've taken the liberty of consolidating all uses of `send_message` (from Mailboxer) into a new service class, that way we have a single place where we need to tie notifications into ActionCable.